tests/integration/kubernetes/gha-run.sh

```diff
@@ -431,6 +436,33 @@ function cleanup_snapshotter() {
 	echo "::endgroup::"
 }
 
+function reset_k8s_images(){
+	# we are encountering the issue (https://github.com/kata-containers/kata-containers/issues/8407)
+	# with containerd on CI is likely due to the content digest being missing from the content store,
+	# which can happen when switching between different snapshotters.
+	# To help sort it out, we now clean up or fetch related images in k8s.io namespace.
+	ctr_args=""
+	if [ "${KUBERNETES}" = "k3s" ]; then
+		ctr_args="--address /run/k3s/containerd/containerd.sock "
+	fi
+	ctr_args+="--namespace k8s.io"
+	ctr_command="sudo -E ctr ${ctr_args}"
+	pause_images=$(${ctr_command} i ls | grep 'pause' | awk '{print $1}')
+	for image in $pause_images; do
+		${ctr_command} content fetch "$image"
+	done
+	# remove related images in k8s.io namespace
+	images_to_remove=(
```

Comment:
I guess these 3 images are test images that you want cleaned up? This seems error prone (what if another test image gets added later and needs this cleanup), but I don't have a better suggestion. Something to consider in case you know of a better way to do this.

Reply:
Yeah, I agree with your suggestion. But currently this issue is likely to arise only with images pulled using a snapshotter other than overlayfs. In this context, I have specifically processed the images used in guest-pull. If there are better suggestions, I would like to make changes.